src/PrestaShopBundle/Utils/BoolParser.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace PrestaShopBundle\Utils;
+
+class BoolParser
+{
+    /**
+     * @param string|int $value
+     *
+     * @return bool
+     */
+    public static function castToBool($value)
+    {
+        if (is_bool($value)) {
+            return $value;
+        }
+        if (is_numeric($value)) {
+            return (bool) $value; // 0 => false; all other true
+        } else {
+            $lValue = strtolower($value);
+
+            return $lValue == 'false' ? false : true;
```

Comment:
```suggestion
            return strtolower($value) === 'true';
```

Reply:
Thanks for the tip :) Actually better would be:
`        return strtolower($value) !== 'false';`

to pass these cases:
```
        $this->assertTrue(BoolParser::castToBool('anything else'));
        $this->assertTrue(BoolParser::castToBool('on'));
```